business/tls_test.go

```diff
@@ -537,7 +538,7 @@ func fakeStrictMeshPeerAuthentication(name string) []*security_v1.PeerAuthentica
 
 func fakeMeshPeerAuthenticationWithMtlsMode(name, mTLSmode string) []*security_v1.PeerAuthentication {
 	mtls := &api_security_v1.PeerAuthentication_MutualTLS{
-		Mode: api_security_v1.PeerAuthentication_MutualTLS_Mode(api_security_v1.PeerAuthentication_MutualTLS_Mode_value[mTLSmode]),
+		Mode: api_security_v1.PeerAuthentication_MutualTLS_Mode(api_security_v1beta1.PeerAuthentication_MutualTLS_Mode_value[mTLSmode]),
```

Comment:
@nrfox what's the reason behind using those enum constants from older api versions? aren't they both constants without locks?

Reply:
Since I updated the istio libraries from 1.22 --> 1.24, these constants no longer exist in `api_security_v1` but they still exist in `api_security_v1beta1`.